From: =?utf-8?q?Timo_R=C3=B6hling?= <timo@gaussglocke.de>
Date: Wed, 2 Dec 2020 14:02:48 +0100
Subject: Fix install location of draco-config.cmake

---
 cmake/draco_install.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/draco_install.cmake b/cmake/draco_install.cmake
index 905bca6..228b701 100644
--- a/cmake/draco_install.cmake
+++ b/cmake/draco_install.cmake
@@ -40,7 +40,7 @@ macro(draco_setup_install_target)
                  "${draco_build}/draco-config.cmake" @ONLY NEWLINE_STYLE UNIX)
   install(
     FILES "${draco_build}/draco-config.cmake"
-    DESTINATION "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DATAROOTDIR}/cmake")
+    DESTINATION "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/cmake/draco")
 
   foreach(file ${draco_sources})
     if(file MATCHES "h$")
