From: =?utf-8?q?Timo_R=C3=B6hling?= <timo@gaussglocke.de>
Date: Wed, 2 Dec 2020 14:02:59 +0100
Subject: Fix removal of build dir prefix from include path

---
 cmake/draco_install.cmake | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cmake/draco_install.cmake b/cmake/draco_install.cmake
index 228b701..9f48b6a 100644
--- a/cmake/draco_install.cmake
+++ b/cmake/draco_install.cmake
@@ -48,12 +48,12 @@ macro(draco_setup_install_target)
     endif()
   endforeach()
 
-  # Strip $draco_src_root from the file paths: we need to install relative to
-  # $include_directory.
-  list(TRANSFORM draco_api_includes REPLACE "${draco_src_root}/" "")
   set(include_directory "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_INCLUDEDIR}")
 
   foreach(draco_api_include ${draco_api_includes})
+    # Strip $draco_src_root from the file paths: we need to install relative to
+    # $include_directory.
+    string(REPLACE "${draco_src_root}/" "" draco_api_include "${draco_api_include}")
     get_filename_component(file_directory ${draco_api_include} DIRECTORY)
     set(target_directory "${include_directory}/draco/${file_directory}")
     install(FILES ${draco_src_root}/${draco_api_include}
